test/pjrt/test_runtime.py

```diff
@@ -55,24 +52,21 @@ def test_xla_device_error(self):
   }, False), ('pjrt_cpu', {
       'PJRT_DEVICE': 'CPU',
       'PJRT_SELECT_DEFAULT_DEVICE': '0'
-  }, True), ('xrt_tpu', {
-      'XRT_TPU_CONFIG': 'localservice;0;localhost:51011'
-  }, False), ('pjrt_tpu_precedence', {
+  }, True), ('pjrt_tpu_precedence', {
       'PJRT_DEVICE': 'TPU',
       'XRT_TPU_CONFIG': 'localservice;0;localhost:51011',
-  }, True), ('xrt_gpu', {
-      'GPU_NUM_DEVICES': '4'
-  }, False), ('pjrt_gpu', {
+  }, True), ('pjrt_gpu', {
       'PJRT_DEVICE': 'GPU',
       'GPU_NUM_DEVICES': '4'
   }, True), ('xla_dist_worker', {
-      'XRT_LOCAL_WORKER': 'c_localservice:2'
-  }, False))
+      'PJRT_DEVICE': 'TPU',
+  }, True))
   def test_pjrt_default_device(self, env_vars, expect_using_pjrt):
     with mock.patch.dict(os.environ, env_vars, clear=True):
       # Print a warningif we had to select a default runtime
       if 'PJRT_DEVICE' not in os.environ and expect_using_pjrt:
         logs_context = self.assertLogs(level=logging.WARNING)
+        print("XRT has been deprecated, please set PJRT_DEVICE")
```

Comment:
Let's refrain from using `print()` calls. can you replace with `assert` calls? 
Also, at some point, these messages are no longer needed to be in the source code (after a few releases). Please level a `TODO` comment to drop them at a future release.

Reply:
Thanks, will delete this print directly since XRT is not mentioned here

will not use assert cause duplicate to the next code checking whether we have log or not in https://github.com/pytorch/xla/blob/1dc615646472fc2154397b1478223ae5283a7927/test/pjrt/test_runtime.py#L79

Reply:
Thanks, updated in https://github.com/pytorch/xla/pull/5202